src/eq_schema/Answer.test.js

```diff
@@ -95,7 +101,7 @@ describe("Answer", () => {
     });
 
     it("should add options even if empty array", () => {
-      const answer = new Answer(createAnswerJSON({ options: [] }));
+      const answer = new Answer(createAnswerJSON({ options: [] }), ctx);
```

Comment:
I don't think you need to pass the `ctx` down into Answer, since it doesn't use it in its constructor. So `ctx` can be removed from each Answer constructor above?